docs/network-mobile/mobile-proof-of-coverage.mdx

```diff
@@ -65,54 +76,73 @@ The proposed new algorithm for MOBILE Reward calculation in the MOBILE Rewards O
 
 1. Supply the declared transmitter power of each Radio and its location to the Obstruction Data
    Oracle.
-2. Get all hexes that have coverage from Outdoor Radios based on the information from the
+2. Get all hexes that have coverage from Outdoor Radios/Access Points based on the information from the
    Obstruction Data Oracle.
 3. Based on the location of Indoor Radios, get all hexes with Indoor coverage and all adjacent
    hexes.
 4. Use projected signal loss information from the Obstruction Data Oracle to determine the potential
    signal strength of each Outdoor Radio in each hex.
-5. For each hex, get at most 5 Outdoor Radios with the top signal strength of the same level.
-
-- If there are more than 5 Radios with the same signal strength level, use the `coverage_claim_time`
-  value to determine the top 5 oldest installations where `coverage_claim_time` is the timestamp
-  when the Radio received the spectrum access grant for the first time.
-- To prevent rewarding "dead" Radios, we propose to reset `coverage_claim_time` if the Radio was not
-  generating a Heartbeat for more than 72 hours and use the time of the last Heartbeat as the new
-  `coverage_claim_time`.
-
-6. Get max 5 Indoor Radios using the same approach as above for Outdoor Radios.
+5. For each hex, get at most 3 Outdoor Radios and 3 Outdoor Wi-Fi Access Points and award the top 3
+   signals based on rank, and provide a decaying multiplier, as noted in the table below. Any with a
+   rank of "Fail" will receive 0 modeled coverage points, and the algorithm will end. Note, the
+   calculation for this will be done in step 10.
+
+| Rank | Hex Limit Multiplier |
+| ---- | -------------------- |
+| 1    | 1.00X                |
+| 2    | 0.50X                |
+| 3    | 0.25X                |
+| Fail | 0.00X                |
+
+
+- If there are more than 1 Radio/Access Point with the same signal strength level, use the
+  `coverage_claim_time` value to rank the top 3 oldest installations where `coverage_claim_time`
+  is the timestamp when the Radio received the spectrum access grant for the first time. The
+  oldest Radio will receive the higher   rank, while the newest radio will receive the lowest rank.
+  To prevent rewarding "dead" Radios,  the `coverage_claim_time` will be reset if the Radio/Access
+  Point was not generating a Heartbeat for more than 72 hours and use the time of the last Heartbeat
```

Comment:
try to avoid capitalizing too many things. Would lowercase 'heartbeat', probably 'radio/access point' too